ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_igmp.schema.yml

```
@@ -12,3 +12,27 @@ keys:
     keys:
       ssm_aware:
         type: bool
+      host_proxy:
+        type: dict
+        keys:
+          vrf:
```

should be "vrfs" plurial
```suggestion
          vrfs:
```

---

ok, i will change the references to vrf in templates too